source/JsMaterialX/JsMaterialXCore/post.js

```diff
@@ -1,10 +1,17 @@
-Module.onRuntimeInitialized = function() {
-  Module.ValueElement.prototype.getValue = function () {
-    const value = Module.ValueElement.prototype._getValue.apply(this);
-    if (value) {
-      return value.getData();
-    }
-    return value;
-  }
-};
+//
```

Comment:
Hum, I didnt add this header to source/JsMaterialX/JsMaterialXFormat/StrContainerTypeRegistration.h, could you add it there as well? the LICENSE thing :)